src/AudioAnalysisTools/AudioAnalysisTools.csproj

```diff
@@ -36,4 +36,7 @@
     </PackageReference>
     <PackageReference Include="Towel" Version="1.0.11-alpha" />
   </ItemGroup>
+  <ItemGroup>
+    <Folder Include="Events\Tracks\" />
+  </ItemGroup>
```

Comment:
This is weird. If I'm reading this right it looks like we've added an empty folder to the solution?

Check the folder is empty, and if it is, remove these lines

Reply:
The folder was empty and I have removed it.